stellargraph/mapper/mini_batch_node_generators.py

```diff
@@ -285,16 +307,9 @@ def __getitem__(self, index):
         # The operations to normalize the adjacency matrix are too slow.
         # Either optimize this or implement as a layer(?)
```

Comment:
Do we still need this comment, or can we re-write to only say that we might be able to improve further as a layer?

Reply:
Ah, yeah, I forgot to update it. 👍